src/qibocal/web/static/styles.css

```
@@ -149,15 +149,15 @@
   border-radius: 15px;
 }
 
-.th_styles
+.div-fitting th
```

```suggestion
.fitting-table th
```

Use the closer parent, stay more local. No need to go beyond.